### DISCUSSION TITLE
Ink/Stitch on a Raspberry Pi

### DISCUSSION THREAD

**Discussion by tyrosinase** [inkstitch/inkstitch#1539] [General]
AHAHAHAHA VICTORY! Set up a new Pi 4 last night, installed Inkscape (from the Raspbian distro) and Ink/Stitch 2.1.0 (from the developer instructions) and it all appears to work.

I still have some tweaking to do before the Pi is fully usable, since it absolutely assumes I have the TV on a desk a few inches from my nose and not that I'm sitting in a recliner a reasonable TV-watching distance away.

![image](https://user-images.githubusercontent.com/9473609/149156080-dde79456-9b1f-454d-9527-46d78d9b9c87.png)

**Comment by OldMateX** [comment]
Congrats! I'd love to see updates, an all in one solution for an embroidery machine like this would be amazing.

**Comment by tyrosinase** [comment]
What I'd really like to figure out is whether the multi-port Pi's can be used in Mass Storage Gadget mode; so far the answer seems no. It doesn't change the convenience much (I have the Zeroes that are connected to the machines mounted as directories on the 4) but it would be nice for folks to only have to buy the one board (especially given the shortage).

So far the 4 is pretty slow compared to my desktop (which is itself not a monster gaming rig or anything) but usable, other than I haven't had a chance to get the trackball set up properly so it's kind of hilarious on the giant screen: it's a long trek from one side of it to the other because it is just not equipped for that kind of size.
